docker-compose.yml

```diff
@@ -1,30 +1,24 @@
 version: '3'
 services:
-  db:
-    build: ./docker_db
-    volumes:
-      - ./docker_db/pgdata:/pgdata
-    ports:
-      - "5432"
-    environment:
-      - POSTGRES_USER=docker
-      - POSTGRES_PASSWORD=docker
-      - PGDATA=/pgdata
+    db:
+        image: postgres:9.6
+        volumes:
+            - ./docker_db/pgdata:/pgdata
+        ports:
+            - "5432"
+        environment:
+            - POSTGRES_USER=docker
+            - POSTGRES_PASSWORD=docker
+            - PGDATA=/pgdata
 
-  backend:
-    # environment:
-    #   - GOPATH=/go/src/atmiguel/cerealnotes/
-    environment:
-      - DATABASE_URL=postgresql://docker:docker@db:5432?sslmode=disable
-      - PORT=8080
-      - TOKEN_SIGNING_KEY=AllYourBase
-    build:
-      context: ./
-      args:
-        app_env: ${APP_ENV} 
-    volumes:
-      - .:/go/src/github.com/atmiguel/cerealnotes/
-    ports:
-      - "8080:8080"
-    links:
```

Comment:
I think having the links/depends on is important. It makes sure docker starts the db image before the server image.

Reply:
Done